perma_web/perma/tasks.py

```diff
@@ -694,18 +686,25 @@ def get_screenshot(link, browser):
         print "Not taking screenshot! %s" % ("Page size is %s." % (page_size,))
         safe_save_fields(link.screenshot_capture, status='failed')
 
+
 def get_page_size(browser):
     try:
-        root_element = browser.find_element_by_tag_name('html')
-    except (NoSuchElementException, URLError):
+        return browser.execute_script("""
+            return {'height': document.body.scrollHeight, 'width': document.body.scrollWidth}
+        """)
```

Comment:
I'm going to spend a few minutes seeing how this js works with phantom running, and see how it works for a few different domains with Chrome running. I didn't really give it a good try Friday.

I'm thinking it might not be that reliable.... maybe something like this instead... https://stackoverflow.com/a/1147768